src/main/java/mate/academy/springboot/datajpa/dto/CategoryResponseDto.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.dto;
+
+public class CategoryResponseDto {
```

Comment:
you use `@Data` on other dto's (which is perfectly fine) but not on this one and a few others
let's use it everywhere possible or not use it at all